constants.py

```diff
@@ -1,57 +1,91 @@
-processed_ff_data_path = os.path.join(conflab_path, 'annotations', 'f_formations', 'processed')
-
-
-# timecode information
-# used to match video timecode with Midge timecode
-vid2_start = datetime(2019, 10, 24, 16, 49, 36, int(1000000 * 58/59.94)) # start timecode of vid2 : 14:49:36:58
-vid3_start = datetime(2019, 10, 24, 17, 7, 13, int(1000000 * 58/59.94)) # start timecode of vid3 : 15:07:13:58
-
-vid_timecodes = {
-    'vid2_seg8': vid2_start + timedelta(minutes=14),
-    'vid2_seg9': vid2_start + timedelta(minutes=16),
-    'vid3_seg1': vid3_start + timedelta(minutes=0),
-    'vid3_seg2': vid3_start + timedelta(minutes=2),
-    'vid3_seg3': vid3_start + timedelta(minutes=4),
-    'vid3_seg4': vid3_start + timedelta(minutes=6),
-    'vid3_seg5': vid3_start + timedelta(minutes=8),
-    'vid3_seg6': vid3_start + timedelta(minutes=10),
-}
+processed_ss_path = os.path.join(
+    conflab_path, "annotations", "actions", "speaking_status", "processed"
+)
+processed_pose_path = os.path.join(conflab_path, "annotations", "pose", "coco")
+processed_wearables_path = os.path.join(conflab_path, "data_processed", "wearables")
+processed_ff_data_path = os.path.join(
+    conflab_path, "annotations", "f_formations", "processed"
+)
+
+RAW_VIDEOS_FRAMERATE: float = 59.94
+
+# timecode information / manually retrieved by the original authors of
+# conflab.
+# Used to match video timecode with Midge timecode
+CAM4_VID2_START_TIMECODE = datetime(
+    2019, 10, 24, 16, 49, 36, int(1000000 * 58 / RAW_VIDEOS_FRAMERATE)
+)  # start timecode of vid2 : 14:49:36:58
+CAM4_VID3_START_TIMECODE = datetime(
+    2019, 10, 24, 17, 7, 13, int(1000000 * 58 / RAW_VIDEOS_FRAMERATE)
+)  # start timecode of vid3 : 15:07:13:58
+
+# All synched audio files start at the same time
+TIMECODE_FOR_ALL_SYNCED_PARTICIPANT_AUDIO_WAV_FILES = datetime.fromtimestamp(1571927168.657)
 
-vid_deltas = {
-    'vid2_seg8': timedelta(minutes=14),
-    'vid2_seg9':  timedelta(minutes=16),
-    'vid3_seg1':  timedelta(minutes=0),
-    'vid3_seg2':  timedelta(minutes=2),
-    'vid3_seg3':  timedelta(minutes=4),
-    'vid3_seg4': timedelta(minutes=6),
-    'vid3_seg5':  timedelta(minutes=8),
-    'vid3_seg6':  timedelta(minutes=10),
+NUMBER_OF_PARTICIPANTS_WITH_WAV_FILE: int = 50
+PARTICIPANTS_IDS_TO_IGNORE: list[int] = [38, 39]
+CAMERAS_OF_INTEREST = [4, 2, 6, 8, 10]
+
+cam4_vid_timecodes: dict[str, datetime] = {
+    "vid2_seg8": CAM4_VID2_START_TIMECODE + timedelta(minutes=14),
+    "vid2_seg9": CAM4_VID2_START_TIMECODE + timedelta(minutes=16),
+    "vid3_seg1": CAM4_VID3_START_TIMECODE + timedelta(minutes=0),
+    "vid3_seg2": CAM4_VID3_START_TIMECODE + timedelta(minutes=2),
+    "vid3_seg3": CAM4_VID3_START_TIMECODE + timedelta(minutes=4),
+    "vid3_seg4": CAM4_VID3_START_TIMECODE + timedelta(minutes=6),
+    "vid3_seg5": CAM4_VID3_START_TIMECODE + timedelta(minutes=8),
+    "vid3_seg6": CAM4_VID3_START_TIMECODE + timedelta(minutes=10),
 }
 
 camera_raw_to_segment = {
```

Comment:
Can you please fix the name of this variable? I picked it in a hurry and never fixed it.
```suggestion
camera_raw_file_number_to_raw_file_name = {
```

Reply:
Definitely agree on renaming. I get confused however on these dictionaries of dictionaries, and I typically end up naming the variable according to the top level dict mapping to the nested dict. Thus: `camera_id_to_camera_raw_file_name_per_video_index` ?

Reply:
I'll use `camera_id_to_dict_of_video_index_to_camera_raw_file`. It feels more natural when using it in scripts

Reply:
Fixed in 124b5be58b35d448a82a245bb968c691206c83f8